src/gui/guiFormSpecMenu.cpp

```diff
@@ -1548,8 +1563,8 @@ void GUIFormSpecMenu::parseImageButton(parserData* data, const std::string &elem
 {
 	std::vector<std::string> parts = split(element,';');
 
-	if ((((parts.size() >= 5) && (parts.size() <= 8)) && (parts.size() != 6)) ||
-		((parts.size() > 8) && (m_formspec_version > FORMSPEC_API_VERSION)))
+	if ((((parts.size() >= 5) && (parts.size() <= 9)) && (parts.size() != 6)) ||
+		((parts.size() > 9) && (m_formspec_version > FORMSPEC_API_VERSION)))
```

Comment:
FYI: Item buttons using the new format won't be displayed at all in MT versions older than this commit.

Reply:
I suppose that can be fixed by changing the second 9 back to an 8? Or is it better to increment FORMSPEC_API_VERSION in this case?

Reply:
Well, the formspec version was never incremented, but would be a a possibility to make it backwards-compatible.

Reply:
I think it's worthwhile. I did a quick test, and incrementing the version seems to more properly display the formspec when the version number is incremented:

- No increment: [Link](https://i.imgur.com/4WncNOi.png)
- No increment: [Link](https://i.imgur.com/pqJuDbe.png)